github.md

```diff
@@ -37,3 +55,54 @@ For example, when configuring [auto-publishing for an npm library](./npm-package
 | production | Repositories that are deployed to production                       |
 
 If none of the above topics fit your need, a PR should be opened to add the new topic to this list before use.
+
+## Repository contents
```

Comment:
Do you think this sits best in a `security` recommendation or here? Maybe link from the security one?

Reply:
These [security](https://github.com/guardian/recommendations/blob/main/security.md#sensitive-data) docs? The information covered in both the docs, whilst related, are different. I think cross linking is a good idea.